Frameworks/Foundation/NSProgress.mm

```diff
@@ -208,7 +209,9 @@ - (void)resignCurrent {
         // If you don’t create any child progress objects between the calls to becomeCurrentWithPendingUnitCount: and resignCurrent,
         // the "parent" progress automatically updates its completedUnitCount by adding the pending units.
         if (!currentProgressStack->top().childCreated) {
-            [self _incrementCompletedUnitCount:currentProgressStack->top().pendingUnitCountToAssign];
+            @synchronized(self) {
+                [self setCompletedUnitCount:self.completedUnitCount + currentProgressStack->top().pendingUnitCountToAssign];
```

Comment:
'basically', but it's a member of the wrapper and not the object

---
In reply to: [95282860](https://github.com/Microsoft/WinObjC/pull/1663#discussion_r95282860) [](ancestors = 95282860)